utils/manifest-generator/generate-llbuild-manifest

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
```

Comment:
Can you see if you can make this work with python2? macOS doesn't have python3 installed by default.